### PR TITLE
[NETCFGX] IDD_TCPIP_BASIC_DLG Radiobuttons need 2 arrow-down-presses instead of one

### PR DESCRIPTION
**Pull request by JoachimHenze**
## Purpose

The 2 Radiobuttons in IDD_TCPIP_BASIC_DLG needed 2 arrow-down-presses to be toggled instead of one.
On the first pressing it even caused a drawing glitch.
The patch is based on a proposal of KrosUser in the Jira-ticket, but I minimized that.
Still he deserves the credit for the first analysis:
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getnextdlggroupitem
Here some Microsoft intern prepared his doctor-thesis describing how it is supposed to work:
> The GetNextDlgGroupItem function searches controls in the order (or reverse order) they were created in the dialog box template. The first control in the group must have the WS_GROUP style; all other controls in the group must have been consecutively created and must not have the WS_GROUP style.When searching for the previous control, the function returns the first control it locates that is visible and not disabled. If the control specified by hCtl has the WS_GROUP style, the function temporarily reverses the search to locate the first control having the WS_GROUP style, then resumes the search in the original direction, returning the first control it locates that is visible and not disabled, or returning hCtl if no such control is found.When searching for the next control, the function returns the first control it locates that is visible, not disabled, and does not have the WS_GROUP style. If it encounters a control having the WS_GROUP style, the function reverses the search, locates the first control having the WS_GROUP style, and returns this control if it is visible and not disabled. Otherwise, the function resumes the search in the original direction and returns the first control it locates that is visible and not disabled, or returns hCtl if no such control is found.If the search for the next control in the group encounters a window with the WS_EX_CONTROLPARENT style, the system recursively searches the window's children.

To sum it up:
* We do have to reorder the controls.
* We do have some missing WS_TABSTOP in some languages
* We do have to strip some dots in some languages

JIRA issue: [CORE-19327](https://jira.reactos.org/browse/CORE-19327)

## Proposed changes
video Before the fix:
https://jira.reactos.org/secure/attachment/66898/66898_0.4.15-dev-6961-g8860dc5_radioButton.webm

video After the fix:
[video.webm](https://github.com/reactos/reactos/assets/33393466/d53c0918-4fa1-4549-a022-087ff6962b42)

## TODO
- [x] Do all the other languages as well
- [ ] do the same for the advanced dialog in future PR

### CODE REVIEW

File: dll/win32/netcfgx/lang/ja-JP.rc [hunk 1]
```diff
@@ -6,17 +6,17 @@ CAPTION "全般"
...
     CONTROL "次の IP アドレスを使う(&U):", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
     LTEXT "IP アドレス:", -1, 14, 75, 135, 8
     CONTROL "", IDC_IPADDR, "SysIPAddress32", WS_TABSTOP, 144, 75 - 2, 85, 12
     LTEXT "サブネット マスク:", -1, 14, 90, 135, 8
     CONTROL "", IDC_SUBNETMASK, "SysIPAddress32", WS_TABSTOP, 144, 90 - 2, 85, 12
     LTEXT "デフォルト ゲートウェイ:", -1, 14, 105, 135, 8
     CONTROL "", IDC_DEFGATEWAY, "SysIPAddress32", WS_TABSTOP, 144, 105 - 2, 85, 12
-    CONTROL "DNS サーバーのアドレスを自動的に取得する", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP, 14, 129, 210, 12
     GROUPBOX "", -1, 9, 147, 228, 47, BS_GROUPBOX
+    CONTROL "DNS サーバーのアドレスを自動的に取得する", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
```
Comment (JoachimHenze) [inline dll/win32/netcfgx/lang/ja-JP.rc:19]: @katahiromz in case this line would end with an unintended dot for ja-JP.rc or any of the 3 chinese languages, then please let me know. I am not so familiar with their characters. And that line should never end with a dot.

**Review by binarymaster — APPROVED** [review]

File: dll/win32/netcfgx/lang/ja-JP.rc (hunk 1 above)
Comment (katahiromz) [reply dll/win32/netcfgx/lang/ja-JP.rc:19]: image
image
image
image
I cannot found any problems.

Comment (JoachimHenze) [reply dll/win32/netcfgx/lang/ja-JP.rc:19]: Thx for your checking. For me it seems as if both radiobuttons which do end with the (U) accelerator should actually end with "(U) :". In those screenshot only the upper one has the ":", but the lower one lacks it. At least both do have the ":" in my german XPSP3 netcfgx.. But I do just mention that here as a fact. I didn't touch that line in my PR, so a fix for that won't be part of my PR neither. I will still leave that discussion-thread in unresolved state.

Comment (katahiromz) [reply dll/win32/netcfgx/lang/ja-JP.rc:19]: Oh, you mean "colon"?
I think it should be added to both.

Comment (katahiromz) [reply dll/win32/netcfgx/lang/ja-JP.rc:19]: But Chinese colon is in fullwidth. Be care.

Comment (JoachimHenze) [reply dll/win32/netcfgx/lang/ja-JP.rc:19]: Initially I really meant unintended dots, like I stripped it here https://github.com/reactos/reactos/pull/6049/commits/e1173341c3742d5c6144045e8a4e85131cd01546 for example.

The thing with the colon I didn't realize before I saw your screenshot (different thing, and different line in the rc).
